docs/Deprecating-Disabling-and-Removing-Formulae.md

```diff
@@ -0,0 +1,110 @@
+# Deprecating, Disabling, and Removing Formulae
+
+There are many reasons why formulae may be deprecated, disabled, or removed. This document explains the differences between each method as well as explaining when one method should be used over another.
+
+## Overview
+
+This general rule of thumb can be followed:
+
+- `deprecate!` should be used for formulae that _should_ no longer be used
+- `disable!` should be used for formulae that _cannot_ be used
+- Formulae that have been disabled for a long time or have a more serious issue should be removed
```

Comment:
Should we add the case of versioned formulae when we have too many and really old ones can be removed?

Reply:
Right, we have a limit defined in the Versioned Formula documentation, that could also be a reason